[WIP] Updated docs content for new editor experience (#194)
stackblitz/docs: docs/guides/user-guide/getting-started.md

# {{ $frontmatter.title }}

<!-- @include: ./parts/environments.md -->

StackBlitz can be used:

> - as an easy way to write and share code without a development environment

Reviewer: Thoughts on this?

```suggestion
- to write and share code with just a url
```
